platform/admin/admin-app/src/main/resources/features.xml

```diff
@@ -16,7 +16,7 @@
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           xsi:schemaLocation="http://karaf.apache.org/xmlns/features/v1.3.0 http://karaf.apache.org/xmlns/features/v1.3.0">
 
-    <feature name="admin-core" install="manual" version="${project.version}"
```

Comment:
These are both still installed as unremovable parts of `admin-app`; they should be left as manual.